src/Engine/GameEvents/HealUnitEvent.java

```diff
@@ -0,0 +1,73 @@
+package Engine.GameEvents;
+
+import Engine.XYCoord;
+import Terrain.MapMaster;
+import UI.MapView;
+import UI.Art.Animation.GameAnimation;
+import Units.Unit;
+
+public class HealUnitEvent implements GameEvent
+{
+  private Unit unit;
+  public final int repairPower = 1;
```

Comment:
Oh. So you aren't using this for turn-init healing. Are you planning to?

Reply:
I built the logic to handle arbitrary healing, but I wasn't gonna rewrangle everything in a branch that was possibly throwaway.

Reply:
So in a word, yes, now that I know that the branch is liable to be merged.

Reply:
Fixed in the `mod` branch